_posts/2020-02-07-metal3-dev-env-install-deep-dive.md

```diff
@@ -0,0 +1,921 @@
+* minikube v1.6.2 on Centos 7.7.1908
+* Selecting 'kvm2' driver from user configuration (alternates: [none])
+```
+
+In the same way as with the host, container images are pre-cached but in this case inside `minikube` local image repository. Notice that in this case the `BareMetal operator` is also downloaded since it will run on `minikube`. The container location is defined by `BAREMETAL_OPERATOR_IMAGE`. In case you want to test new features or new fixes to the `BareMetal operator`, just change the value of the variable to match the location of the modified image:
+
+<br>
+<br>
+
+| **Name of the variable** | **Default value**      |
+| BAREMETAL_OPERATOR_IMAGE | quay.io/metal3-io/baremetal-operator |
+
+<br>
+> **Note:** Remember that `minikube` is the management cluster in our environment. So it must run all the operators and controllers needed for Metal³
+<br>
+<br>
+
+```sh
++ sudo su -l -c 'minikube ssh sudo docker image ls' alosadag
+REPOSITORY                                TAG                 IMAGE ID            CREATED             SIZE
+quay.io/metal3-io/ironic                  latest              e5d81adf05ee        26 hours ago        693MB
+quay.io/metal3-io/ironic-ipa-downloader   latest              d55b0dac2144        6 days ago          239MB
+quay.io/metal3-io/ironic-inspector        latest              8bb5b844ada6        6 days ago          408MB
+quay.io/metal3-io/baremetal-operator      latest              3c692a32ddd6        9 days ago          1.77GB
+k8s.gcr.io/kube-proxy                     v1.17.0             7d54289267dc        7 weeks ago         116MB
+k8s.gcr.io/kube-controller-manager        v1.17.0             5eb3b7486872        7 weeks ago         161MB
+k8s.gcr.io/kube-scheduler                 v1.17.0             78c190f736b1        7 weeks ago         94.4MB
+k8s.gcr.io/kube-apiserver                 v1.17.0             0cae8d5cc64c        7 weeks ago         171MB
+kubernetesui/dashboard                    v2.0.0-beta8        eb51a3597525        7 weeks ago         90.8MB
+k8s.gcr.io/coredns                        1.6.5               70f311871ae1        2 months ago        41.6MB
+k8s.gcr.io/etcd                           3.4.3-0             303ce5db0e90        3 months ago        288MB
+kubernetesui/metrics-scraper              v1.0.2              3b08661dc379        3 months ago        40.1MB
+k8s.gcr.io/kube-addon-manager             v9.0.2              bd12a212f9dc        6 months ago        83.1MB
+k8s.gcr.io/pause                          3.1                 da86e6ba6ca1        2 years ago         742kB
+gcr.io/k8s-minikube/storage-provisioner   v1.8.1              4689081edb10        2 years ago         80.8MB
+```
+
+Once container images are stored, `minikube` can be stopped. In that moment, the virtual networks shown in the previous picture are attached to the `minikube` VM as can be verified by the following command: 
```

Comment:
Once the container images are stored, minikube can be stopped. In that moment, the virtual networks shown in the previous picture are attached to the minikube VM as it can be verified by the following command: